schedules/utils/schedules.py

```diff
@@ -164,8 +165,10 @@ def send_given_shift_email(shift_slot):
         <p>Du har blitt satt opp på vakt!</p>
         <p>Vakt: {shift_slot.shift.name}</p>
         <p>Hvor: {shift_slot.shift.location}</p>
+        <p>Når: {shift_slot.shift.datetime_start.strftime('%d.%m kl %H:%M')} - {shift_slot.shift.datetime_end.strftime('%H:%M')}</p>
+
         
-            """
+        """
```

Comment:
Should probably cast this to local time I use this approach for interview emails.

```python
def notify_interviewers_cancelled_interview_email(applicant, interview):
    local_time = timezone.localtime(
        interview.interview_start, pytz.timezone(settings.TIME_ZONE)
    )
    name = applicant.get_full_name
    interview_location = interview.location.name
    formatted_local_time = local_time.strftime("%d.%m.%Y kl. %H:%M")
    content = (
```

Reply:
Skal ta en titt på det!

Reply:
Castet til local time nu, endret ikke format som sendes i mail tho